sentry/src/main/java/io/sentry/ISpan.java

```diff
@@ -118,10 +134,29 @@
    */
   void setTag(@NotNull String key, @NotNull String value);
 
+  @Nullable
+  String getTag(@NotNull String key);
+
   /**
    * Returns if span has finished.
    *
    * @return if span has finished.
    */
   boolean isFinished();
+
+  /**
+   * Returns if transaction is sampled.
+   *
+   * @return is sampled
+   */
+  @Nullable
+  Boolean isSampled();
+
+  /**
+   * Returns the last active span within the span hierarchy
+   *
+   * @return the last active span of {@code null} if not found
+   */
+  @Nullable
+  ISpan getLatestActiveSpan();
```

Comment:
why would we need this in the ISpan interface?

Reply:
Either this or we use `instance of` in Scope:

```
public ISpan getLatestActiveSpan() {
  final ISpan tx = span;
  if (tx instanceof SentryTracer) {
    return ((SentryTracer) tx).getLatestActiveSpan();
  } else {
    return tx;
  }
}
```

Reply:
When I look at it now, as much as I don't like using `instanceof` i think in this case it will be a lesser evil.